src/CMakeLists.txt

```diff
@@ -40,9 +40,8 @@ set(HEADERS
             logging.h
             openssl_compat.h
             ostree.h
-            ostreefake.h
-            ostreeinterface.h
```

Comment:
Nice. I didn't understand  why  ostreeinterface.h is needed.

Reply:
I had grand plans that I realized were pointless, that's all!